TYPO3.Flow/Tests/Unit/Http/Component/TrustedProxiesComponentTest.php

```diff
@@ -440,6 +440,12 @@ public function forwardHeaderTestsDataProvider()
                 'requestUri' => 'https://acme.com',
                 'expectedUri' => 'https://acme.com:80',
             ),
+            array(
```

Comment:
Not if the other declarations use the long form. And since it's better not to mix functional and cleanup, let's leave it like that.